bgpd/bgp_vty.c

```diff
@@ -321,22 +321,22 @@ int bgp_vty_find_and_parse_afi_safi_bgp(struct vty *vty,
 		else {
 			*bgp = bgp_lookup_by_name(vrf_name);
 			if (!*bgp) {
-				use_json
-					? vty_out(vty, "{}\n")
-					: vty_out(vty,
-						  "View/Vrf specified is unknown: %s\n",
-						  vrf_name);
+				vty_out(vty,
+					use_json
+						? "{}\n"
+						: "View/Vrf specified is unknown: %s\n",
+					vrf_name);
```

Comment:
Hm, I guess this doesn't generate a warning about the leftover vrf_name parameter? I don't think it'll break anything... :thinking:

Reply:
I guess I need to quit relying on clang-format to do the right thing.  This is what how it did the formatting.

Reply:
Okay, the mystery is at least solved why clang-format isn't working correctly. The .clang-format now included apparently disabled the length check. I've removed that file and clang-format is trying to make things fit correctly.  Will make the changes and resubmit.

Reply:
(btw with the warning i was referring to `"{}\n"` not having a `%s` in there)

Reply:
So much for that.  Repeating the issuing of clang-format doesn't show what I thought I was seeing.  Also Quentin says to ignore the warning since okay for string constants to be beyond 80 columns and checkpatch still complains.  Quentin also says the vrf_name location appears to be correct. At this point, would you prefer I go back to a if/then construct instead of the conditional operator?

Reply:
Ah, misunderstood the comment (obviously).  How would you like me to proceed?

Reply:
I'm thinking I should at least change that location back to if/then because the vrf_name %s difference.  Agree?

Reply:
Decided to change all of these from conditional operators to if/then for consistency and making checkpatch happier.    Will push that change soon.